src/contracts/core/StrategyManagerStorage.sol

```diff
@@ -56,17 +56,6 @@ abstract contract StrategyManagerStorage is IStrategyManager {
     mapping(address => uint256) public numWithdrawalsQueued;
     /// @notice Mapping: strategy => whether or not stakers are allowed to deposit into it
     mapping(IStrategy => bool) public strategyIsWhitelistedForDeposit;
-    /*
-     * @notice Mapping: staker => virtual 'beaconChainETH' shares that the staker 'owes' due to overcommitments of beacon chain ETH.
-     * When overcommitment is proven, `StrategyManager.recordOvercommittedBeaconChainETH` is called. However, it is possible that the
-     * staker already queued a withdrawal for more beaconChainETH shares than the `amount` input to this function. In this edge case,
-     * the amount that cannot be decremented is added to the staker's `beaconChainETHSharesToDecrementOnWithdrawal` -- then when the staker completes a
-     * withdrawal of beaconChainETH, the amount they are withdrawing is first decreased by their `beaconChainETHSharesToDecrementOnWithdrawal` amount.
-     * In other words, a staker's `beaconChainETHSharesToDecrementOnWithdrawal` must be 'paid down' before they can "actually withdraw" beaconChainETH.
-     * @dev In practice, this means not passing a call to `eigenPodManager.withdrawRestakedBeaconChainETH` until the staker's 
-     * `beaconChainETHSharesToDecrementOnWithdrawal` has first been reduced to zero.
-    */
-    mapping(address => uint256) public beaconChainETHSharesToDecrementOnWithdrawal;
```

Comment:
might need to make this an empty slot to preserve the storage layout
perhaps something like
```
   // unused storage slot(s) reserved to preserve storage layout
   uint256[1] internal _unusedStorageSlots;
```

ignore + resolve this if there's no storage layout changes here

Reply:
Fixed! Good catch